sdk/webpubsub/azure-messaging-webpubsub/src/main/java/module-info.java

```diff
@@ -1,11 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
 
 module com.azure.messaging.webpubsub {
     requires transitive com.azure.core;
 
-    requires com.nimbusds.jose.jwt;
-
     exports com.azure.messaging.webpubsub;
-    exports com.azure.messaging.webpubsub.models;
```

Comment:
Hi @samvaity . I saw the lines in module-info.java are still being removed. But I tried using autorest.java 4.1.22 locally to generate, these lines are not removed. Please use the latest autorest.java  to try and let me know if there is any issue.
![image](https://github.com/Azure/azure-sdk-for-java/assets/87355844/524d74c2-db4d-4d1e-aa74-ee49900fe87e)

Reply:
Hey, I still don't see the module-info changes being reverted with 4.1.22

Reply:
Hey @samvaity , partial-update compares existing code with the generated code, please make sure you have added back the lines to `module-info.java` before running autorest command, otherwise partial-update does not have the existing code to compare with.
![image](https://github.com/Azure/azure-sdk-for-java/assets/87355844/7ae5907a-bae2-40b2-9de1-47f2ca40b02d)